tools/src/main/java/com/nedap/archie/flattener/Flattener.java

```diff
@@ -169,7 +169,7 @@ public Archetype flatten(Archetype toFlatten) {
             prohibitZeroOccurrencesConstraints(result);
         }
 
-        rulesFlattener.combineRules(child, result, "prefix", "", "", true /* override statements with same tag */);//TODO: actually set a unique prefix
+        rulesFlattener.combineRules(child, result, "", "", "", true /* override statements with same tag */);//TODO: actually set a unique prefix
```

Comment:
You removed the "prefix", why?

Also it might be an even better idea than before to set a unique prefix here, maybe that should be done as well?

Reply:
I've added a unique prefix. I am not really sure whether this is needed. Note that a prefix here will make the rules less readable since it will in some cases lead to double-prefixing.